CHANGELOG.md

```diff
@@ -102,16 +53,35 @@
 
 * Removes length limits around partition names
 
+* Removes dependencies on the existence of public schema
+
 * Executor avoids opening extra connections
 
+* Excludes orphaned shards while finding shard placements
+
+* Fixes a bug that allowed moving of shards belonging to a reference table
+
 * Fixes a bug that can cause a crash when DEBUG4 logging is enabled
 
-* Fixes data race in `get_rebalance_progress`
+* Fixes a bug that causes a crash when a column default includes type casting
+  (e.g. nextval('seq_name'::text))
```

Comment:
I am unsure if this is something that we introduced in 10.1

Maybe #5046 fixed a bug introduced in #4914

Reply:
Yes, I think so. We should probably remove this entry